scripts/helper/create-cluster.sh

```diff
@@ -11,110 +11,56 @@ ARG1=$1
-		OCP_RELEASE="4.8.57"
-		RHCOS_VERSION="4.8"
-		if [ -z "$RHCOS_RELEASE" ]; then
-			RHCOS_RELEASE="4.8.14"                   # Latest release of RHCOS 4.8 at this time
-		fi
-		RHCOS_SUFFIX="-$RHCOS_RELEASE"
-		;;
-	4.9)
-		OCP_RELEASE="4.9.59"
-		RHCOS_VERSION="4.9"
-		if [ -z "$RHCOS_RELEASE" ]; then
-			RHCOS_RELEASE="4.9.45"                   # Latest release of RHCOS 4.9 at this time
-		fi
-		RHCOS_SUFFIX="-$RHCOS_RELEASE"
-		;;
-	4.10)
-		OCP_RELEASE="4.10.67"
-		RHCOS_VERSION="4.10"
-                if [ -z "$RHCOS_RELEASE" ]; then
-                        RHCOS_RELEASE="4.10.37"                   # Latest release of RHCOS 4.10 at this time
-                fi
-                RHCOS_SUFFIX="-$RHCOS_RELEASE"
-		;;
-	4.11)
-                OCP_RELEASE="4.11.59"
-                RHCOS_VERSION="4.11"
-                if [ -z "$RHCOS_RELEASE" ]; then
-                        RHCOS_RELEASE="4.11.48"                   # Latest release of RHCOS 4.11 at this time
-                fi
-                RHCOS_SUFFIX="-$RHCOS_RELEASE"
-                ;;
+
 	4.12)
-		OCP_RELEASE="4.12.55"
+		OCP_RELEASE="4.12.60"
                 RHCOS_VERSION="4.12"
                 if [ -z "$RHCOS_RELEASE" ]; then
                         RHCOS_RELEASE="4.12.30"                   # Latest release of RHCOS 4.12 at this time
                 fi
                 RHCOS_SUFFIX="-$RHCOS_RELEASE"
 		;;
         4.13)
-                OCP_RELEASE="4.13.40"
+                OCP_RELEASE="4.13.45"
                 RHCOS_VERSION="4.13"
                 if [ -z "$RHCOS_RELEASE" ]; then
                         RHCOS_RELEASE="4.13.10"                   # Latest release of RHCOS 4.13 at this time
                 fi
                 RHCOS_SUFFIX="-$RHCOS_RELEASE"
                 ;;
 	4.14)
-                OCP_RELEASE="4.14.21"
+                OCP_RELEASE="4.14.33"
                 RHCOS_VERSION="4.14"
                 if [ -z "$RHCOS_RELEASE" ]; then
                         RHCOS_RELEASE="4.14.15"                   # Latest release of RHCOS 4.14 at this time
                 fi
                 RHCOS_SUFFIX="-$RHCOS_RELEASE"
                 ;;
         4.15)
-                OCP_RELEASE="4.15.9"
+                OCP_RELEASE="4.15.22"
                 RHCOS_VERSION="4.15"
                 if [ -z "$RHCOS_RELEASE" ]; then
                         RHCOS_RELEASE="4.15.0"                   # Latest release of RHCOS 4.14 at this time
                 fi
                 RHCOS_SUFFIX="-$RHCOS_RELEASE"
                 ;;	
-	4.16)
-                unset OCP_RELEASE
+        4.16)
+                OCP_RELEASE="4.16.3"
                 RHCOS_VERSION="4.15"
+                if [ -z "$RHCOS_RELEASE" ]; then
+                        RHCOS_RELEASE="4.16.0"                   # Latest release of RHCOS 4.14 at this time
+                fi
+                RHCOS_SUFFIX="-$RHCOS_RELEASE"
+                ;;
+
+	4.17)
+                unset OCP_RELEASE
+                RHCOS_VERSION="4.17"
```

Comment:
Here RHCOS_VERSION should be 4.16
@Shilpi-Das1

Reply:
Changed it